tests/Doctrine/DBAL/Migrations/Tests/Stub/Functional/MigrateNotTouchingTheSchema.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Doctrine\DBAL\Migrations\Tests\Stub\Functional;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class MigrateNotTouchingTheSchema extends AbstractMigration
```

Comment:
Should be implemented as spy: we want to verify that all methods are actually being called with our mock schema